src/borg/archiver/extract_cmd.py

```diff
@@ -56,33 +55,50 @@ def do_extract(self, args, repository, manifest, archive):
         else:
             pi = None
 
-        for item in archive.iter_items(filter):
-            archive.preload_item_chunks(item, optimize_hardlinks=True)
+        for item in archive.iter_items(None):
```

Comment:
Isn't "None" the default anyway?

Reply:
Yes, you're right. I thought passing None would make the code's behavior more explicit and clear. I can remove it to simplify the code.